server/game/cards/07-GR/Webstur.js

```diff
@@ -0,0 +1,36 @@
+const Card = require('../../Card.js');
+
+class Webstur extends Card {
+    // After Fight: For each damage on Webstur, you may discard the top card
+    // of each player's deck.
+    setupCardAbilities(ability) {
+        this.fight({
+            gameAction: ability.actions.sequentialForEach((context) => ({
+                num: context.source.tokens.damage,
```

Comment:
Let's say Webstur has 3 damage, but we only want to discard one card, we have to click No twice, right?

I wonder if it is better to provide option:`Discard` and `Stop`, instead of no.

Reply:
> Let's say Webstur has 3 damage, but we only want to discard one card, we have to click No twice, right?

Correct.  That is abiding by a literal reading of the card ability.

> I wonder if it is better to provide option:Discard and Stop, instead of no.

Sure, that would be a valid way to do it too I think.  If you prefer that way, let me know and I'll try it.

Reply:
Not sure we would implement the "Stop" part, but I think it is better.

Reply:
@romanogit please take another look.  I tried using a 'may' ability instead and tracking the number of iterations in a variable.  When the user chooses "No", it stops.

Reply:
You rock! 🌟